includes/global-settings-komoju.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+if (!defined('ABSPATH')) {
+    exit;
+}
+
+/*
+ * Settings for the whole Komoju plugin
+ */
+
+return [
+    [
+        'title'       => 'API Settings',
+        'type'        => 'title',
+        'id'          => 'api-settings-in-komoju',
+        'description' => sprintf(__('Default url for the webhook is %s. Use this if you\'re not sure what it should be.', 'komoju-woocommerce'), $this->url_for_webhooks()),
+    ],
+    [
+        'id'          => 'komoju_woocommerce_secret_key',
+        'placeholder' => 'sk_live_000000000000000000000000',
+        'title'       => __('Secret Key from Komoju', 'komoju-woocommerce'),
+        'type'        => 'text',
+        'default'     => get_option('woocommerce_komoju_settings')['secretKey'],
```

Comment:
These 'default' keys pull in the value from the payment gateway setting. This way, even if an existing user upgrades their KOMOJU plugin, there's no need to re-enter every field.